rep/processors/CtGovPdfProcessor.py

```diff
@@ -44,17 +57,27 @@ def _process_blob_wrapper(self, blob):
             [x[0] for x in vote_list]
         )
 
-
     def _get_page_from_blob(self, blob):
         fileReader = PyPDF2.PdfFileReader(io.BytesIO(blob))
         page_content = fileReader.getPage(0).extractText()
         page_content = page_content.replace('\n', '')
         page_content += '\n'
         return page_content
 
-
-    def _write_to_csv(self, year, date, bill_number, vote_name, rep_name, rep_vote):
+    def _write_to_db(self, year, date, bill_number, vote_name, rep_name, rep_vote):
+        # with open('bills.csv', 'a') as f:
         for num, _ in enumerate(rep_vote):
             logging.info(f"""
                 Year: {year}, Date: {date}, Bill Number: {bill_number}, Vote Name: {vote_name}
                 Rep Name: {rep_name[num]}, Rep Vote: {rep_vote[num]}""")
+            # row = [year, date, bill_number, vote_name, rep_name[num], rep_vote[num]]
```

Comment:
remove commented out code, unless you want this to always create a csv, in which case, just uncomment it

Reply:
👍